integration_tests/assets/etc/wazo-amid/conf.d/50-default.yml

```diff
@@ -9,7 +11,16 @@ ajam:
   username: wazo_amid
   password: opensesame
 
+ami:
+  host: asterisk-ami
+  username: wazo_amid
+  password: opensesame
+
 rest_api:
   listen: 0.0.0.0
 
-publish_ami_events: False
+bus:
+  host: rabbitmq
+  username: guest
+  password: guest
+  startup_connection_delay: 1
```

Comment:
startup_connection_delay => "How many seconds between each try to connect to the bus at startup".

The param is required, because used in bus/client.py

value=1 in integration tests, because this is the value used config.py/config.yml